modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -279,6 +323,72 @@ class TestFunctional
         }
     }
 
+    template <typename T>
+    inline void initPointRandU(cv::RNG& rng, T& pt)
+    { ::initPointRandU(rng, pt); }
+
+    // initialize std::vector<cv::Point_<T>>/std::vector<cv::Point3_<T>>
+    template <typename T, template <typename> class Pt>
+    void initPointsVectorRandU(const int sz_in, std::vector<Pt<T>> &vec_)
+    {
+        cv::RNG& rng = theRNG();
+
+        vec_.clear();
+        vec_.reserve(sz_in);
+
+        for (int i = 0; i < sz_in; i++)
+        {
+            Pt<T> pt;
+            initPointRandU(rng, pt);
+            vec_.push_back(pt);
```

Comment:
And here is warning too:
```
gapi_tests_common.hpp(343): warning C4702: unreachable code
```

Reply:
Mb `emplace_back`, or wrapping `(Pt{pt})` might help